Library/Homebrew/download_strategy.rb

```diff
@@ -15,6 +15,7 @@ def initialize(name, resource)
     @url = resource.url
     @version = resource.version
     @meta = resource.specs
+    extend Pourable if meta.key?(:bottle) && meta[:bottle]
```

Comment:
When would `meta[:bottle]` be truthy but not `meta.key?(:bottle)` (I think never, no?)

Reply:
Ah thanks, I switch languages a lot, I thought I was protecting against an exception on accessing a non-existent key.

Reply:
No worries, we've all been there!